### PR TITLE
staging to main

### PR DESCRIPTION
**Pull request by truemiller**
- new staking contracts integration
- health checker fixes
- utf8 middleware encoding
- improved seed phrase creation UI

released as [rc184](https://github.com/valory-xyz/olas-operate-app/releases/tag/untagged-9e76efff1d0a96127c72)

### CODE REVIEW

File: frontend/constants/contractAddresses.ts
Comment (truemiller) [inline frontend/constants/contractAddresses.ts:1]: please review

File: frontend/constants/stakingProgramMeta.ts
Comment (truemiller) [inline frontend/constants/stakingProgramMeta.ts:1]: please review @mohandast52

File: frontend/context/StakingContractInfoProvider.tsx
Comment (truemiller) [inline frontend/context/StakingContractInfoProvider.tsx:1]: same here, removing manual addition requirement

File: frontend/utils/service.ts [hunk 1]
```diff
@@ -18,6 +19,18 @@ export const getMinimumStakedAmountRequired = (
...
+  if (stakingProgramId === StakingProgramId.Beta3) {
+    return 100;
+  }
+
+  if (stakingProgramId === StakingProgramId.Beta4) {
+    return 100;
+  }
+
+  if (stakingProgramId === StakingProgramId.Beta5) {
+    return 10;
```
[the remark this answers is not in the document]
Comment (truemiller) [reply frontend/utils/service.ts:31]: yep, on govern, it is 5 OLAS, and we do `n * agentInstances + 1` where `agentInstances === 1`

**Review by iasonrovis — APPROVED** [review]

**Review by jmoreira-valory — APPROVED** [review]

**Pull request merged**